hknweb/tours/models.py

```diff
@@ -0,0 +1,32 @@
+from django.db import models
+from django.utils import timezone
+from django.conf import settings
+# from django.contrib.admin.widgets import AdminDateWidget
+
+MAX_STRLEN = 255 # default max length for char fields
+MAX_TXTLEN = 2000 # default max length for text fields
+
+#.save adds to database
+# POST user submits
+#GET user wants something
+class DepTour(models.Model):
+    """
+    Model for a department tour
+    """
+
+    class Meta:
+        verbose_name = "Department Tour"
+
+    name            = models.CharField(max_length=MAX_STRLEN, default='')
+    email           = models.EmailField(max_length=MAX_STRLEN, default='')
+    # verify_email    = models.EmailField(max_length=MAX_STRLEN, default='')
```

Comment:
Why did we comment out this code? It's not too particularly necessary, but it's definitely good to have, especially if in the future we want to send the requester a confirmation email or something. Probably ties in to [this](https://github.com/compserv/hknweb/pull/189/files?file-filters%5B%5D=.lock&file-filters%5B%5D=.py&file-filters%5B%5D=No+extension#diff-c36ae4f1c5ec9bda34ddbd1926b73238R18) too, so I'll include them in the same comment.